rdrf/rdrf/templates/rdrf_cdes/alert_language_is_not_supported.html

```diff
@@ -0,0 +1,26 @@
+{% load get_language_settings_codes %}
+{% get_language_settings_codes as LANGUAGES %}
+
+<script type="text/javascript">
+    $(document).ready(function(){
+        const browser_language = navigator.language;
+        const browser_language_in_settings = [browser_language, browser_language.split('-')[0]].some((language) => {{ LANGUAGES|safe }}.includes(language));
+        const existing_not_support_language = localStorage.getItem('not_support_language');
+        const $alertPanel = document.getElementById('id_language_not_support_alert');
+        $alertPanel.innerHTML = `Language ${browser_language} is not currently supported.`
```

Comment:
Needs to be translated. You can use the `gettext` javascript method to help with this.